src/features/click-to-play.js

```diff
@@ -2144,7 +2226,8 @@ async function createYouTubePreview (originalElement, widget) {
     topSection.appendChild(titleElement)
 
     /** Text Button on top section */
-    const textButton = makeTextButton(widget.replaceSettings.buttonText, widget.getMode())
+    // Use darkMode styles because of preview background
```

Comment:
Maybe explain why? E.g. "Use darkMode styles because video preview background can make button text hard to read otherwise"? (Same below.)